app/components/PostModalContent.tsx

```
@@ -0,0 +1,50 @@
+        </div>
+        <div className={styles.postRight}>
+          <p className={styles.postInfo}>
+            <span className={styles.postAuthorName}>{post.author?.name}</span>
+            <span className={styles.postTimeAndSourceFeed}>
+              {timeAgoLabel}
+              {post.feed ? ` in ${post.feed.name}` : ""}
+            </span>
+          </p>
+          <div
+            className={styles.postContent}
+            dangerouslySetInnerHTML={{ __html: post.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)